src/assets/scripts/client/airline/AirlineController.js

```diff
@@ -39,6 +54,7 @@ export default class AirlineController {
      * @param icao {string}
      */
     addAirline(icao) {
+        console.error('.addAirline() is a deprecated method');
```

Comment:
Usually, yes, I do.  In this case, though, this method is getting called as a result of previous functionality that should no longer be used.  I throw the `console.error` in there for two reasons: one, I see the annoying red errors when the method gets hit and, two, I also see the stack trace of what methods were called to get here.  It helps in re-routing the stream, to use the analogy we talked about earlier.